README.md

```diff
@@ -18,31 +20,46 @@ The overall steps to get to a running model are:
 
 1. Install this `neural-lam` package. See [full installation instructions](#installing-neural-lam) below.
 2. Process any data you want to work with to a training-optimized format using `mllam-data-prep`. For example, to process the DANRA data run:
+
     ```
     python -m mllam_data_prep scripts/danra_interior_config.yaml
     ```
+
     Such config files for DANRA, COSMO, ERA5 and IFS data are all available in the `scripts` repository.
 3. Build the graph for the model using
+
     ```
     python -m neural_lam.build_rectangular_graph
     ```
+
     or
+
     ```
     python -m neural_lam.build_triangular_graph
     ```
+
     See the scripts `scripts/danra_build_graphs.sh` and `scripts/cosmo_build_graphs.sh` for commands used to build the graphs from the paper.
 4. Train or evaluate the model. For both eval and training use
+
     ```
     python -m neural_lam.train_model
     ```
+
     with different options. Use
+
     ```
     python -m neural_lam.train_model --help
     ```
+
     for a full list of options available.
     The scripts `scripts/danra_eval.sh` and `scripts/cosmo_eval.sh` contain full configs with options that allow for loading the final model checkpoints from the paper.
 
+### Reproduce COSMO Example (Step‑by‑Step)
+
+  For a practical, end‑to‑end walkthrough (download → preprocess → graph → train/eval), see the concise guide in `docs/reproduce_paper_sample.md`.
```

Comment:
Could reference both reproduction docs (both full and sample) here

Reply:
done https://github.com/joeloskarsson/neural-lam-dev/pull/15/commits/e121d1361116379d2ecd5285bf4c8cdeebf191d7